samples/openapi3/client/petstore/dart-dio/petstore_client_lib_fake/lib/src/api/fake_api.dart

```diff
@@ -10,7 +10,6 @@ import 'package:dio/dio.dart';
 import 'dart:typed_data';
 import 'package:built_collection/built_collection.dart';
 import 'package:openapi/src/api_util.dart';
-import 'package:openapi/src/model/date.dart';
```

Comment:
did this import get removed since it is now a primitive? did you try to build the result dart project?
```
flutter pub get
flutter pub run build_runner build
```

Reply:
Yea, this is a result of adding them as primitives.